src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,122 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import jdk.nashorn.internal.objects.annotations.Constructor;
+
+@Entity
+public class Book {
+
+    @Column()
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private int pages;
```

Comment:
The type of pages is Integer, according to the card

Reply:
Isn't this type what the card is referred to? Or maybe the correct data type is "Integer", isn't it?

Reply:
First, in the card the type is Integer.
Second, in Java is not the same int that Integer. Int is a type primitive, and integer is an object, and this brings its consequences, for example if i receive an int, i can modify it to my liking, and it should not happen.
For more information, i leave this link, you check an answer correct
https://es.stackoverflow.com/questions/54419/diferencia-entre-int-e-integer